engine-tests/src/main/java/org/terasology/HeadlessEnvironment.java

```diff
@@ -291,14 +276,11 @@ protected void setupModuleManager(Set<Name> moduleNames) throws Exception {
         EntitySystemSetupUtil.addReflectionBasedLibraries(context);
     }
 
-    /**
-     * @throws IOException ShrinkWrap errors
-     */
     @Override
     protected void setupPathManager() throws IOException {
-        final JavaArchive homeArchive = ShrinkWrap.create(JavaArchive.class);
-        final FileSystem vfs = ShrinkWrapFileSystems.newFileSystem(homeArchive);
-        PathManager.getInstance().useOverrideHomePath(vfs.getPath(""));
+        Path tempHome = Files.createTempDirectory("terasology-env", (FileAttribute<?>[]) null);
+        tempHome.toFile().deleteOnExit();
+        PathManager.getInstance().useOverrideHomePath(tempHome);
```

Comment:
I had to double-check to make sure this isn't a HeadlessEnvironment used by servers.

It's in `engine-tests`, so I _think_ it is separate.

Reply:
yep it is separate